diceware/__init__.py

```diff
@@ -193,6 +196,8 @@ def get_passphrase(options=None):
     result = options.delimiter.join(words)
     for _ in range(options.specials):
         result = insert_special_char(result, rnd=rnd)
+    if options.max > 0:
+        result = result[:options.max]
```

Comment:
Shouldn't truncating happen _before_ inserting special chars (that might get lost this way round)?